charts/camunda-platform/templates/zeebe/statefulset.yaml

```diff
@@ -251,8 +251,17 @@ spec:
       spec:
         accessModes: {{ .Values.zeebe.pvcAccessModes }}
         {{- if .Values.zeebe.pvcStorageClassName }}
+        {{- if (eq "-" .Values.zeebe.pvcStorageClassName) }}
```

Comment:
No need to make special handling here, using `null` as a value should have the same effect.

Reply:
It was added to follow bitnami standard. But if you want it removed I can change it

Reply:
We need some handling here else the default storage class is used, which is not the desired outcome in cases where one wants to use pre provisioned PVs, hence not use storage classes.

Reply:
https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_storage.tpl

Reply:
any thoughts @aabouzaid

Reply:
did you read what @kristeey wrote @aabouzaid? This function will help us use pre-created disks in Azure (Probably the same for GCP and AWS). With this functionality we can the take down the entire environment and spin it up again after bigger maintenance tasks and continue running with the same disks when we spin up the new AKS cluster. 

This is partly because Azure configure the PV disks in a resource group that will dissapear when we delete an AKS cluster.

We also have a support ticket for this [SUPPORT-22209](https://jira.camunda.com/servicedesk/customer/portal/1/SUPPORT-22209)

Reply:
Thanks for the info, I will take a look into it again soon.